src/main/scala/ai/privado/semantic/Language.scala

```diff
@@ -56,17 +56,44 @@ object Language {
 
   }
   implicit class StepsForPropertyForSqlQueryNode(val trav: Traversal[SqlQueryNode]) extends AnyVal {
-    def file: Traversal[File] = trav.out(EdgeTypes.SOURCE_FILE).cast[File]
-
+    def file: Traversal[File]             = trav.out(EdgeTypes.SOURCE_FILE).cast[File]
+    def sqlTable: Traversal[SqlTableNode] = trav.out(EdgeTypes.AST).cast[SqlTableNode]
   }
 
   implicit class StepsForPropertyForSqlTableNode(val trav: Traversal[SqlTableNode]) extends AnyVal {
-    def file: Traversal[File] = trav.out(EdgeTypes.SOURCE_FILE).cast[File]
-
+    def file: Traversal[File]               = trav.out(EdgeTypes.SOURCE_FILE).cast[File]
+    def sqlColumn: Traversal[SqlColumnNode] = trav.out(EdgeTypes.AST).cast[SqlColumnNode]
   }
 
   implicit class StepsForPropertyForSqlColumnNode(val trav: Traversal[SqlColumnNode]) extends AnyVal {
     def file: Traversal[File] = trav.out(EdgeTypes.SOURCE_FILE).cast[File]
 
   }
+
+  implicit class SqlColumnObject(val node: SqlColumnNode) {
+    def sqlTable: Option[SqlTableNode] = {
```

Comment:
Can we simply not write `node.start.in(EdgeTypes.AST).cast[SqlTableNode]` ?

Reply:
Addressed separately here: 203d820542cb72558c426009549788d534356d4a
@pandurangpatil please also review this commit.

Reply:
@khemrajrathore I guess the suggested code snippet is same as `node.in(EdgeTypes.AST).cast[SqlTableNode].head`. I didn't understood the purpose of `start` in this statement. 

In the event there is no parent node and if it returns an empty `Iterator` calling `.head` will throw error on it.

Reply:
@pandurangpatil Ideally we should avoid using `head` in this case as well and the return should be a traversal instead of a single node. 

Change the return type from `Option[SqlTableNode]` to `Traversal[SqlTableNode]` and similarly for other helper traversals as well

Cc - @hiteshbedre

Reply:
@khemrajrathore it is expected to have only one parent to the given `SqlColumn` node.

what is the reasoning behind not calling `head` at this place as we know there is only a possibility of a single return node.

Either we do that at this place or we do it where we are going to consume this method. I would rather handle it at his place than doing it again and again everytime I consume this method.

Reply:
@pandurangpatil Yes, the expectation is to get 1 parent, but in some cases if a sqlColumn node is created such that it has no parent of multiple parent (Due to some faulty implementation or bug) , that should also work smoothly and not throw runtime exception. 

And if you also refer to the core Language engine query they to refer to the same principle of using Traversal as much as possible. Not a blocker from my end its all about making queries robust

Reply:
Also we want to do `head` instead of `Some(node.start.in(EdgeTypes.AST).cast[SqlTableNode].head)` we should do `node.start.in(EdgeTypes.AST).cast[SqlTableNode].headOption`

cc - @hiteshbedre

Reply:
> @khemrajrathore I guess the suggested code snippet is same as `node.in(EdgeTypes.AST).cast[SqlTableNode].head`. I didn't understood the purpose of `start` in this statement.
> 
> In the event there is no parent node and if it returns an empty `Iterator` calling `.head` will throw error on it.

@pandurangpatil Won't this lead of `nullPointerException` if there is no parent ?

Reply:
@khemrajrathore I agree with what you are pointing at when we are using the query like `cpg.sqlColumn.sqlTable.l` in that case it should be an iterator as you are pointing at. As that's going to be called a query.

In this case, the method is going to be called over an Object of `SqlColumnNode` and not on `Traversal[SqlColumnNode]`. If we do that over `Traversal[SqlColumnNode]` then certainly we need to handle it the way you are pointing.

Reply:
> > @khemrajrathore I guess the suggested code snippet is same as `node.in(EdgeTypes.AST).cast[SqlTableNode].head`. I didn't understood the purpose of `start` in this statement.
> > In the event there is no parent node and if it returns an empty `Iterator` calling `.head` will throw error on it.
> 
> @pandurangpatil Won't this lead of `nullPointerException` if there is no parent ?

That's what the null check is being done in the original code.

Reply:
@hiteshbedre Have discussed this with @pandurangpatil and have confirmed the query needs to be 
`node.in(EdgeTypes.AST).collectAll[SqlTableNode].headOption` please make appropriate change and confirm.